README.md

```diff
@@ -1 +1,28 @@
-# dotnet-foundation
\ No newline at end of file
+# Dotnet Foundation
+
+## Overview of the project
+
+Dotnet Foundation is a boiler plate project for ASP .NET projects. This boiler plate can be used in any .NET project for Osmosys. This project contains common files that  can be reused  again and again. Objective of building this project is to make it so that people can copy the project and build on top of it.
```

Comment:
- boiler plate -> boilerplate
- ASP .NET -> ASP.NET.
- Dotnet Foundation is a boilerplate for a ASP.NET WebAPI project.
- Remove "again and again".

Reply:
made all the rectifications as given.